system_tests/language.py

```diff
@@ -115,3 +115,10 @@ def test_analyze_entities_from_blob(self):
         document = Config.CLIENT.document_from_url(gcs_url)
         entities = document.analyze_entities()
         self._check_analyze_entities_result(entities)
+
+    def test_analyze_sentiment(self):
+        positive_msg = 'Jogging is fun'
+        document = Config.CLIENT.document_from_text(positive_msg)
+        sentiment = document.analyze_sentiment()
+        self.assertEqual(sentiment.polarity, 1)
+        self.assertTrue(0.5 < sentiment.magnitude < 1.5)
```

Comment:
We can also leave it as-is, just knowing that we are subject to the whims of the back-end and adjust if/when it breaks (maybe a comment to that effect?)